actions/team_actions.test.js

```diff
@@ -1,15 +1,14 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import configureStore from 'redux-mock-store';
-import thunk from 'redux-thunk';
-
 import * as TeamActions from 'mattermost-redux/actions/teams';
 import * as channelActions from 'mattermost-redux/actions/channels';
 import * as userActions from 'mattermost-redux/actions/users';
 
 import * as Actions from 'actions/team_actions.jsx';
 
+import store from 'tests/test_store';
```

Comment:
don't we need to import the mock here as well?

Reply:
What do you mean by the mock? If you mean `redux-mock-store`, we have access to it on this object since that's what `tests/test_store` exports. `test_store` doesn't pretend that it's result is just a regular store

Reply:
I think you might have a typo, i was wondering why the import here is named differently than other places. The default export func is called mockStore xD

Reply:
Oh, I see. Yeah, I definitely messed up the name here. I'll fix that